src/TootleLib/makefile

```diff
@@ -1,31 +1,31 @@
-CC 		= g++ -fpermissive -msse -D_SOFTWARE_ONLY_VERSION -D_LINUX
+CC      = g++ -fpermissive -msse -D_SOFTWARE_ONLY_VERSION -D_LINUX
```

Comment:
Please revert all the changes in this file as makefiles do require TAB instead of spaces.